smach_config:
  setup_sequence: # actions to run once at the beginning
    - localize: 'start_localization' # spin around and get tf (from aruco)
    - move: 'start_center' # face the aruco marker dead on to get best reading
    #- sense: 'sense_rocks' # sense rocks and add them to costmap
  loop_sequence: # actions to keep looping on 
    - move: 'nav_to_dig'
    - move: 'nav_to_bin'
    - move: 'nav_to_other'
  
  # Definitions of actions
  mining_params:
    # Positions to move to relative to the collector bin (0,0)
    moves:
      start_center:
        position:
          x: 1.5
          y: 0.0
        # degree off the x-axis. positive is counter-clockwise
        deg: 180.0 
      nav_to_dig:
        position:
          x: 6.0
          y: 0.0
        # degree off the x-axis. positive is counter-clockwise
        deg: 0.0 
      nav_to_bin:
        position:
          x: 1.0
          y: 0.0
        # degree off the x-axis. positive is counter-clockwise
        deg: 0.0 
      nav_to_other:
        position:
          x: 3.0
          y: 0.0
        # degree off the x-axis. positive is counter-clockwise
        deg: 0.0 
